lang/cs/Org.Apache.REEF.IMRU/API/IMRUJobDefinition.cs

```diff
@@ -66,6 +67,7 @@ public sealed class IMRUJobDefinition
         /// <param name="updateTaskMemory">Update task memory</param>
         /// <param name="mapTaskCores">Number of map task cores</param>
         /// <param name="updateTaskCores">Number of update task cores</param>
+        /// <param name="maxRetryNumberInRecovery">Max retry number in recovery</param>
```

Comment:
Can you make comment more informative  like: "Number of times driver will try to re-run IMRU in case some evaluator fails".